target/linux/mpc85xx/files/arch/powerpc/boot/dts/hiveap-330.dts

```diff
@@ -53,17 +53,8 @@
 				};
 
 				partition@40000 {
-					reg = <0x40000 0x40000>;
-					label = "initrd";
-				};
-
-				partition@80000 {
-					reg = <0x80000 0x27c0000>;
-					label = "rootfs";
-				};
-
-				partition@2840000 {
-					reg = <0x2840000 0x800000>;
+					compatible = "openwrt,uimage", "denx,uimage";
+					reg = <0x40000 0x3000000>;
```

Comment:
Is there a reason to keep the `stock-jffs2` partition around given that this is essentially a one-way conversion? Leaving 14.75MB of space unused seems a bit silly. 

This partition gets formatted/initialized by the OEM firmware if it fails to mount, too, so even if someone _did_ go and revert to OEM firmware it wouldn't be an issue.

For that matter, `dtb` probably doesn't need to be defined as a separate partition either since that's going to be prepended to the kernel by the build system & abstracted away from users. It's convenient to have it separate for dev/test when switching out device trees, but that's not something we do for other targets in general AFAIK?

[here's the DTS i've come up with containing those changes and a few quality of life improvements](https://github.com/neg2led/openwrt/blob/2a48e12890b29537852f143ffe277438a1b07e1d/target/linux/mpc85xx/files/arch/powerpc/boot/dts/hiveap-330.dts) - thoughts?

Reply:
Yes, you got it in your edit!

The stock-jffs2 can probably go; this would increase the JFFS2 format time to just under 7 minutes.

I am compiling now to test; I want to verify the JFFS2 isn't scanned e.g. by u-boot.

Reply:
The `stock-jffs2` is now removed as per force-push 3a111004ddf3.